src/Migration/MessageQueue/Handler/MigrationProcessorRegistry.php

```diff
@@ -25,8 +25,12 @@ public function __construct(private readonly iterable $processors)
     {
     }
 
-    public function getProcessor(MigrationStep $step): MigrationProcessorInterface
+    public function getProcessor(MigrationStep $step): ?MigrationProcessorInterface
```

Comment:
I know Lars created https://github.com/shopware/shopware/issues/12625 to document all the breaking changes that happened in our feature branch, but also what prevents us from already getting started with the changes we do now?

Would be great if this break is added in `UPGRADE.md`, otherwise we have to do it later in the above mentioned ticket